README.md

```diff
@@ -20,17 +20,22 @@ Repository to describe, develop, document and test the NumberVerification API fa
 
 ## Meetings
 
-* Meetings are held virtually in Zoom
-* Current schedule & meeting links: [Meetings information](documentation/MeetingMinutes/README.MD)
+* Current schedule, registration, & meeting links are available on the confluence page: [Meetings information](https://wiki.camaraproject.org/display/CAM/NumberVerification)
+
 
-## Results
-* Sub Project is in progress
 
 ## Status and released versions
 
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
-* **The latest available release and version of CAMARA NumberVerification API is 0.3.1. This is the first alpha version of the API.** There are bug fixes to be expected and incompatible changes in upcoming releases. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
-* Release 0.3.1 of the API is available within the [release-0.3.1 branch](https://github.com/camaraproject/NumberVerification/tree/release-0.3.1):
+* **The latest available release and version of CAMARA Number Verification API is 1.0.0-rc.1 This is the first stable version candidate of the API**. It is suitable for implementors and could be use with customers in productive environments.
```

Comment:
```suggestion
* **The latest pre-release of CAMARA Number Verification API is 1.0.0-rc.1 This is the release candidate of the first stable version of the API**. It is suitable for implementors.
```
Only the public version v1.0.0 should be used in production. The release candidate is for implementors only.